service/ogp/parser/parser.go

```diff
@@ -34,7 +34,15 @@ func ParseMetaForURL(url *url.URL) (*opengraph.OpenGraph, *DefaultPageMeta, erro
 	client := http.Client{
 		Timeout: 5 * time.Second,
 	}
-	resp, err := client.Get(url.String())
+
+	req, err := http.NewRequest("GET", url.String(), nil)
+	if err != nil {
+		return nil, nil, ErrNetwork
+	}
+
+	req.Header.Add("user-agent", `bot`)
```

Comment:
user-agentはアクセスしたものが何者なのかを相手に知らせる目的もあるので、可能なら既にあるこちらがわと合わせられると嬉しいですね
https://github.com/traPtitech/traQ/blob/8a2db60a416f32cf022a74c4bd795d6a4c468b84/service/ogp/parser/domain_vrchat.go#L57

Reply:
書き直さなくても、同じパッケージなので単純に変数を参照できます
```suggestion
	req.Header.Add("User-Agent", userAgent)
```

Reply:
Twitterがbotという文字列が入っていないとOGPを返さないようなので、少し名前を変えました

Reply:
元の変数を書き換えて、コメントしておくとよいかもですね